ansible_ai_connect/ai/api/model_pipelines/http/pipelines.py

```diff
@@ -140,7 +140,7 @@ def self_test(self) -> Optional[HealthCheckSummary]:
         )
         try:
             headers = {"Content-Type": "application/json"}
-            r = requests.get(self.config.inference_url + "/readiness", headers=headers)
+            r = requests.get(self.config.inference_url + "/readiness", headers=headers, timeout=5)
```

Comment:
The pipeline configuration has a timeout property. Did you want to consider using that? Or did you prefer a dedicated timeout for self test?

Reply:
The two timeouts target two different things. Here we are just ensure the service is alive, and a 1s timeout should just fine.